packages/react-components/.storybook/main.js

```diff
@@ -4,8 +4,18 @@ module.exports = {
   },
   stories: ['../lib/**/*.stories.mdx', '../lib/**/*.stories.@(js|jsx|ts|tsx)'],
   addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
-  webpackFinal: (config) => {
-    config.resolve.extensions = ['.ts', '.tsx', '.mjs', '.js', '.jsx', '.cjs'];
-    return config;
+  webpackFinal: async (config) => {
+    return {
+      ...config,
+      resolve: {
+        ...config.resolve,
+        extensions: ['.ts', '.tsx', '.mjs', '.js', '.jsx', '.cjs'],
+        alias: {
+          ...config.resolve.alias,
+          '@emotion/core': require.resolve('@emotion/react'),
+          'emotion-theming': require.resolve('@emotion/react'),
```

Comment:
https://github.com/mui-org/material-ui/issues/24282#issuecomment-796755133
https://mui.com/guides/migration-v4/#storybook-emotion-with-v5

We need to let storybook use the latest version of emotion so that the theme gets picked up correctly.

Reply:
Can you add a FIXME explaining this?